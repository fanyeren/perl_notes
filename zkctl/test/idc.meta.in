#!/bin/bash
source $(dirname $0)/common

node_clear idc

raw='"auto_start_new_service":"off","safe_mode":"off"'
./zkctl -w idc add --idc-id=jx --data='{'$raw'}'
echo "add result $?"
./zkctl -w idc get-meta-attr --idc-id=jx --key=auto_start_new_service
echo "get result=$?"
./zkctl -w idc set-meta-attr --idc-id=jx --key=auto_start_new_service --value=on
echo "set result=$?"
./zkctl -w idc get-meta-attr --idc-id=jx --key=auto_start_new_service
echo "get result=$?"
