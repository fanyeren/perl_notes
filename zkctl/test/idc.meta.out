add result 0
off
get result=0
set result=0
on
get result=0
